pkg/scanner/scanner.go

```diff
@@ -300,6 +334,22 @@ func (s *Scanner) Run(ctx context.Context) {
 	defer runtime.HandleCrash()
 	defer s.queue.ShutDown()
 
+	gcsClient, err := storage.NewClient(ctx)
+	if err != nil {
+		klog.Errorf("Failed to create GCS client: %v", err)
+		return
+	}
+	s.gcsClient = gcsClient
+	defer gcsClient.Close()
+
+	metricClient, err := monitoring.NewMetricClient(ctx)
+	if err != nil {
+		klog.Errorf("Failed to create metric client: %v", err)
+		return
+	}
+	s.metricClient = metricClient
+	defer metricClient.Close()
+
```

Comment:
Why don't we initialize these clients in NewScanner? Similar to other clients like kubeClient?

Reply:
Yea the only difference is we hit the error during driver initialization rather than later during Run, but I think there's pros and cons to both.